build.sbt

```diff
@@ -19,7 +19,15 @@ lazy val core: Project = (project in file("core"))
     libraryDependencies ++= Seq(
       "com.propensive" %% "magnolia" % "0.11.0",
       scalatestDependency % "test",
-    )
+    ),
+    unmanagedSourceDirectories in Compile += {
+      // sourceDirectory returns a platform-scoped directory, e.g. /.jvm
+      val sourceDir = (baseDirectory in Compile).value / "src" / "main"
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, n)) if n >= 13 => sourceDir / "scala-2.13+"
+        case _                       => sourceDir / "scala-2.13-"
```

Comment:
if one is +, one is -, I wouldn't know where 2.13 sources go :)

Reply:
Any suggestions for that? I am not sure if changing  `scala-2.13+` to `scala-2.13+=` will do the job.

Reply:
I see I have the same in quicklens, must be a copy-paste ;) I guess I would do scala-2.12 and scala-2.13+, but that's not really that important